internal/proto/daemon.proto

```diff
@@ -18,7 +18,8 @@ message Service {
     string dockerfile_text = 2; // base, e.g. no code
     repeated Mount mounts = 3;
     repeated Cmd steps = 4;
-    string dockerfile_tag = 5; // e.g. gcr.io/blorgly/blorgly-dev
+    Cmd entrypoint = 5;
```

Comment:
I'm pretty sure protobuf doesn't want you to change the meaning of a field after you add it. If the fifth field in a message is a string, it should always be a string. This is so definitions can be backwards compatible. So in this case we could instead add `Cmd entrypoint = 6` and leave `dockerfile_tag`.

Is that true @nicks?

Reply:
yeahhhh i know but that seems more relevant after the API is like, shipped to the world and is in use. i figure when we're still all internal we can change the proto at will for the sake of cleanliness...?

Reply:
I won't die on this hill but I think it's worth practicing being correct. I actually would have expected the proto compiler to fail though, so maybe I'm wrong.

Reply:
ya, never change the tag numbers. this is a good habit to get into.

the tag numbers don't necessarily need to be in ascending order; it's perfectly reasonable to do

```
Cmd entrypoint = 6;
string dockerfile_tag = 5; 
```

if field organization is what you're concerned about